ci_support/run_docker_build.sh

```diff
@@ -70,4 +70,11 @@ yum install -y devtoolset-2-gcc-gfortran
     set +x
     conda build /recipe_root --quiet || exit 1
     upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+
+    set -x
+    export CONDA_NPY=110
```

Comment:
Just a whole bunch of warnings were fixed, no breakage: 
https://github.com/numpy/numpy/pull/8391 
https://github.com/numpy/numpy/pull/8359
https://github.com/numpy/numpy/pull/8178